Sources/FirebaseAndroid/include/module.modulemap

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+module FirebaseAndroid {
+  header "FirebaseAndroid.h"
```

Comment:
I think we should add `export *` ?

Reply:
Hmm, is there a reason to aggressively re-export everything?

Reply:
I think it produces better typed experience in Swift, for example `JNIEnv *` would become just OpaquePointer in Swift withon `export`. Perhaps that's ok, but it's also easier then to make mistakes in Swift when just using OpaquePointer types.

Reply:
Okay, fair enough. I'll add the `export *`.